cunumeric/module.py

```diff
@@ -2323,6 +2367,60 @@ def choose(a, choices, out=None, mode="raise"):
     return a.choose(choices=choices, out=out, mode=mode)
 
 
+@add_boilerplate("c", "a")
+def compress(c, a, axis=None, out=None):
+    """
+    Return selected slices of an array along given axis.
+
+    When working along a given axis, a slice along that axis is returned
+    in output for each index where condition evaluates to True.
+    When working on a 1-D array, compress is equivalent to extract.
```

Comment:
Should "extract" be pointing to another function?

Reply:
what do you mean?

Reply:
"Extract" is a reference to another cunumeric function. Should it be pointing to the documentation for that function?